src/main/java/com/musseukpeople/woorimap/tag/infrastructure/QueryTagRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.musseukpeople.woorimap.tag.infrastructure;
+
+import java.util.List;
+
+import com.musseukpeople.woorimap.couple.domain.Couple;
+import com.musseukpeople.woorimap.tag.application.dto.TagRequest;
+import com.musseukpeople.woorimap.tag.entity.Tag;
+
+public interface QueryTagRepository {
+    List<Tag> findExistTagByCoupleId(
+        Couple couple,
+        List<TagRequest> tagList);
```

Comment:
```suggestion
    List<Tag> findExistTagByCoupleId(Couple couple, List<TagRequest> tagList);
```

한 줄로 보면 기분 좋을 거 같아요~~